src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TaxStatType.java

```diff
@@ -75,245 +75,255 @@ enum TaxStatType {
+    NUM_NECSTATETAXWITHH_DV_CHK_STUB_EXCLUSIONS_DETERMINED("message.tax.num.necstatetaxwithh.dv.chk.stub.exclusions.determined", true),
     NUM_NECSTATETAXWITHH_DV_CHK_STUB_EXCLUSIONS_DETERMINED_DV("message.tax.num.necstatetaxwithh.dv.chk.stub.exclusions.determined.dv"),
     NUM_NECSTATETAXWITHH_DV_CHK_STUB_EXCLUSIONS_DETERMINED_PDP("message.tax.num.necstatetaxwithh.dv.chk.stub.exclusions.determined.pdp"),
-    NUM_NECSTATETAXWITHH_DV_CHK_STUB_NEITHER_DETERMINED("message.tax.num.necstatetaxwithh.dv.chk.stub.neither.determined"),
+    NUM_NECSTATETAXWITHH_DV_CHK_STUB_NEITHER_DETERMINED("message.tax.num.necstatetaxwithh.dv.chk.stub.neither.determined", true),
     NUM_NECSTATETAXWITHH_DV_CHK_STUB_NEITHER_DETERMINED_DV("message.tax.num.necstatetaxwithh.dv.chk.stub.neither.determined.dv"),
     NUM_NECSTATETAXWITHH_DV_CHK_STUB_NEITHER_DETERMINED_PDP("message.tax.num.necstatetaxwithh.dv.chk.stub.neither.determined.pdp"),
-    NUM_NECSTATEINCOME_DV_CHK_STUB_INCLUSIONS_DETERMINED("message.tax.num.necstateincome.dv.chk.stub.inclusions.determined"),
+    NUM_NECSTATEINCOME_DV_CHK_STUB_INCLUSIONS_DETERMINED("message.tax.num.necstateincome.dv.chk.stub.inclusions.determined", true),
     NUM_NECSTATEINCOME_DV_CHK_STUB_INCLUSIONS_DETERMINED_DV("message.tax.num.necstateincome.dv.chk.stub.inclusions.determined.dv"),
     NUM_NECSTATEINCOME_DV_CHK_STUB_INCLUSIONS_DETERMINED_PDP("message.tax.num.necstateincome.dv.chk.stub.inclusions.determined.pdp"),
-    NUM_NECSTATEINCOME_DV_CHK_STUB_EXCLUSIONS_DETERMINED("message.tax.num.necstateincome.dv.chk.stub.exclusions.determined"),
+    NUM_NECSTATEINCOME_DV_CHK_STUB_EXCLUSIONS_DETERMINED("message.tax.num.necstateincome.dv.chk.stub.exclusions.determined", true),
     NUM_NECSTATEINCOME_DV_CHK_STUB_EXCLUSIONS_DETERMINED_DV("message.tax.num.necstateincome.dv.chk.stub.exclusions.determined.dv"),
     NUM_NECSTATEINCOME_DV_CHK_STUB_EXCLUSIONS_DETERMINED_PDP("message.tax.num.necstateincome.dv.chk.stub.exclusions.determined.pdp"),
-    NUM_NECSTATEINCOME_DV_CHK_STUB_NEITHER_DETERMINED("message.tax.num.necstateincome.dv.chk.stub.neither.determined"),
+    NUM_NECSTATEINCOME_DV_CHK_STUB_NEITHER_DETERMINED("message.tax.num.necstateincome.dv.chk.stub.neither.determined", true),
     NUM_NECSTATEINCOME_DV_CHK_STUB_NEITHER_DETERMINED_DV("message.tax.num.necstateincome.dv.chk.stub.neither.determined.dv"),
     NUM_NECSTATEINCOME_DV_CHK_STUB_NEITHER_DETERMINED_PDP("message.tax.num.necstateincome.dv.chk.stub.neither.determined.pdp");
 
     public final String propKey;
+    public final boolean hasTaxSourceSpecificSubStat;
 
-    private TaxStatType(String propKey) {
+    private TaxStatType(final String propKey) {
+        this(propKey, false);
+    }
+
+    private TaxStatType(final String propKey, final boolean hasTaxSourceSpecificSubStat) {
         this.propKey = propKey;
+        this.hasTaxSourceSpecificSubStat = hasTaxSourceSpecificSubStat;
+    }
+
+    public String getPropKey() {

```

Comment:
is this needed?  I thought we were leaning to the convention of directly accessing the enum final variables?

Reply:
Ok, I'll remove this method and just use the immutable field instead.